src/SFA.DAS.FAT.Web/Views/CourseProviders/CourseProviders.cshtml

```diff
@@ -109,8 +125,15 @@
                                 @await Html.PartialAsync("_trainingOptions", new TrainingOptionsViewModel
                                 {
                                     IsEmployerLocationAvailable = provider.IsEmployerLocationAvailable,
+                                    NearestEmployerLocation = provider.NearestEmployerLocation,
                                     IsDayReleaseAvailable = provider.IsDayReleaseAvailable,
-                                    IsBlockReleaseAvailable = provider.IsBlockReleaseAvailable
+                                    IsDayReleaseMultiple = provider.IsBlockReleaseMultiple,
+                                    NearestDayRelease = provider.NearestDayRelease,
+                                    IsBlockReleaseAvailable = provider.IsBlockReleaseAvailable,
+                                    IsBlockReleaseMultiple = provider.IsBlockReleaseMultiple,
+                                    NearestBlockRelease = provider.NearestBlockRelease,
+                                    Distance = Model.Distance,
+                                    Location = Model.Location
                                 })
```

Comment:
Can this model be initialised in code by adding this as a property to the parent view model